position_constraint:
  link_name: "tool0"
  tolerance: 0.1
orientation_constraint:
  link_name: "tool0"
  tolerances: [0.1, 0.1, 0.1]
joint_constraint_shoulder_pan:
  joint_name: "shoulder_pan_joint"
  angle_deg: 90.0
  tolerance_deg: 60.0
joint_constraint_wrist1:
  joint_name: "wrist_1_joint"
  angle_deg: -125.0
  tolerance_deg: 60.0
